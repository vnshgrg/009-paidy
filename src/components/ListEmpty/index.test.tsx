import "react-native";
import { Text } from "react-native";
import React from "react";
import { ListEmpty } from "./index";

import renderer from "react-test-renderer";

it("renders ListEmpty correctly", () => {
  const tree = renderer.create(<ListEmpty />).toJSON();
  expect(tree).toMatchSnapshot();
});
