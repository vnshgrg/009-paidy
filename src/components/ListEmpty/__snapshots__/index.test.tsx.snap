// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ListEmpty correctly 1`] = `
<View
  style={
    {
      "marginVertical": 100,
    }
  }
>
  <Text
    style={
      {
        "color": "#999",
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  >
    No todo items yet.
  </Text>
</View>
`;
